lib/trade_tariff_frontend.rb

```diff
@@ -24,6 +24,7 @@ def accessible_api_endpoints
       geographical_areas
       chemical_substances
       simplified_procedural_code_measures
+      exchange_rates
```

Comment:
I have removed this from accessible_api_endpoints and left it in public_api_endpoints because we need that to allow file downloading. There are upcoming tickets to expose the API and documentation will be added with those